src/main/java/net/atos/zac/flowable/FlowableService.java

```diff
@@ -56,6 +57,8 @@
 @Transactional
 public class FlowableService {
 
+    private static final String VAR_CASE_NIET_ONTVANKELIJK_TOELICHTING = "nietOntvankelijkToelichting";
```

Comment:
Deze moet dus public zijn en kan dan hergebruikt worden in UpdateZaakLifecycleListener

Reply:
Aangepast